samples/config.sh

```diff
@@ -0,0 +1,10 @@
+API_KEY="<api-key>"  #Api key value
+RHEL_URL="https://access.cdn.redhat.com/content/origin/files/sha256/4e/4e5c7cccb5ea7622640c97725912222458af743cf994e48f10239734c6a2ee65/rhel-8.3-ppc64le-kvm.qcow2" #Rhel url
+RHCOS_URL="https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.6/latest/rhcos-4.6.1-ppc64le-openstack.ppc64le.qcow2.gz" #Rhcos url
```

Comment:
when using latest folder use the file without the release suffix - "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.6/latest/rhcos-openstack.ppc64le.qcow2.gz"

Reply:
Removed the URL and added it as example